CHANGES.rst

```diff
@@ -335,7 +335,9 @@ astropy.wcs
 Other Changes and Additions
 ---------------------------
 
-
+- The page in the documentatin describing the astropy docstring rules has now
```

Comment:
I don't think we need any changelog  for this.

Reply:
typo: "documentation" (if the changelog entry stays there :))